axis.go

```diff
@@ -171,6 +171,18 @@ func (LogScale) Normalize(min, max, x float64) float64 {
 	return (log(x) - logMin) / (log(max) - logMin)
 }
 
+// InvertedScale can be used as the value of an Axis.Scale function to
+// invert the axis using any Normalizer. This can be useful for plots that
```

Comment:
The second sentence here should be deleted, I think. It's not adding information

Reply:
👍